src/python/grpcio_tests/tests/interop/resources.py

```diff
@@ -14,27 +14,25 @@
 """Constants and functions for data used in interoperability testing."""
 
 import argparse
+import pkgutil
 import os
 
-import pkg_resources
 
 _ROOT_CERTIFICATES_RESOURCE_PATH = 'credentials/ca.pem'
 _PRIVATE_KEY_RESOURCE_PATH = 'credentials/server1.key'
 _CERTIFICATE_CHAIN_RESOURCE_PATH = 'credentials/server1.pem'
 
 
 def test_root_certificates():
-    return pkg_resources.resource_string(__name__,
-                                         _ROOT_CERTIFICATES_RESOURCE_PATH)
+    return pkgutil.get_data('tests.interop', _ROOT_CERTIFICATES_RESOURCE_PATH)
```

Comment:
Using the `__name__` seems better than hard coding the module name.

Reply:
I've now switched back to `__name__`.